insights/parsers/installed_product_ids.py

```diff
@@ -88,23 +88,37 @@ class InstalledProductIDs(CommandParser):
             Brand Type:
             Brand Name:
 
-    Filters have been added to the parser so that only the ``ID`` element will
-    be collected.
+    Filters have been added to the parser so that only the filtered lines will be collected.
 
     Attributes:
         ids (set): set of strings of the unique IDs collected by the command
+        product_certs(list): list of dicts of the product certificates key-value pairs
 
     Examples:
         >>> type(products)
         <class 'insights.parsers.installed_product_ids.InstalledProductIDs'>
         >>> list(products.ids)
         ['69']
+        >>> products.product_certs[0]
+        {'Path': '/etc/pki/product-default/69.pem', 'ID': '69'}
 
     """
     def parse_content(self, content):
         """ Parse command output """
         self.ids = set()
+        self.product_certs = []
+        one_file_data = None
         for line in content:
-            if line.strip().startswith('ID:'):
-                _, id = line.strip().split(':', 1)
-                self.ids.add(id.strip())
+            # different file delimiter
+            if line == 'Product Certificate':
+                if one_file_data:
+                    self.product_certs.append(one_file_data)
+                one_file_data = {}
+            else:
```

Comment:
When there are no ":" in `line`, this following `split()` will run into exception, eg: 
```
>>> name, value = "some line".split(':', 1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: not enough values to unpack (expected 2, got 1)
```

What about adding the checking of  `line` inside this `else` clause ?
```suggestion
            elif line and ":" in line:
```